npm/npm_import.bzl

```diff
@@ -209,33 +114,19 @@ def npm_translate_lock(
 
         npm_package_lock: The `package-lock.json` file written by `npm install`.
 
-            Only one of [npm_package_lock, yarn_lock] may be set.
+            Only one of `npm_package_lock` and `yarn_lock` may be set.
```

Comment:
nit: the "and" feels out of place

I'd go with: Only one of `npm_package_lock`, `yarn_lock` may be set.

Random example with that form: https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/transport_sockets/tls/v3/secret.proto#extensions-transport-sockets-tls-v3-secret

or: Only one of `npm_package_lock` or `yarn_lock` may be set.

Reply:
I like the last option